src/compile/scale.ts

```diff
@@ -19,15 +19,10 @@ import {Model} from './model';
 import {smallestUnit} from '../timeunit';
 import {UnitModel} from './unit';
 
-/**
- * Color Ramp's scale for legends.  This scale has to be ordinal so that its
- * legends show a list of numbers.
- */
-export const COLOR_LEGEND = 'color_legend';
-
-// scale used to get labels for binned color scales
-export const COLOR_LEGEND_LABEL = 'color_legend_label';
-
+// Scale suffix for scale used to get drive binned legends.
```

Comment:
Why not /** */ ? 

If you use /** */, you can mouseover and get tooltip in VSCode too.  :)